lib/Core/Executor.cpp

```diff
@@ -3398,6 +3398,18 @@ void Executor::terminateStateOnError(ExecutionState &state,
   Instruction * lastInst;
   const InstructionInfo &ii = getLastNonKleeInternalInstruction(state, &lastInst);
 
+  if (termReason == Executor::Abort && CheckMemCleanup) {
```

Comment:
Do we want the first part of the condition? I guess we want to report this for any termination reason.

Reply:
You're right, we want to report it for any termination reason, not just abort. I'll fix this.